app/models/server/models/Messages.js

```diff
@@ -614,6 +614,17 @@ export class Messages extends Base {
 		return this.update(query, update);
 	}
 
+	setEphemeralTime(rid, time) {
+		console.log('inside message model');
+		const query = { rid };
```

Comment:
If no room I'd is provided, this could apply the expiration to all messages, please check the param before passing it to the query